doc/man7/migration_guide.pod

```diff
@@ -0,0 +1,1814 @@
+
+See L</Deprecated low-level encryption functions>.
+IDEA, MD2, MD4 and MDC2 have been moved to the L<Legacy Provider|/Legacy Algorithms>.
+
+=item IDEA_options and MD2_options
+
+These functions return constant strings and are no longer supported.
+
+=item MD5, MD5_Init, MD5_Update, MD5_Final and MD5_Transform
+
+See L</Deprecated low-level encryption functions>.
+
+=item NCONF_WIN32
+
+This undocumented function has no replacement.
+See L<config(5)/HISTORY> for more details.
+
+=item OCSP_parse_url()
+
+Use L<OSSL_HTTP_parse_url(3)> instead.
+
+=item OCSP_REQ_CTX type and OCSP_REQ_CTX_*() methods.
+
+These methods were used to collect all necessary data to form a HTTP request,
+and to perform the HTTP transfer with that request.  With OpenSSL 3.0, the
+type is OSSL_HTTP_REQ_CTX, and the deprecated functions are replaced
+with OSSL_HTTP_REQ_CTX_*(). See L<OSSL_HTTP_REQ_CTX(3)> for additional details.
+
+=item OPENSSL_fork_child, OPENSSL_fork_parent and OPENSSL_fork_prepare
+
+There is no replacement for these functions. These pthread fork support methods
+were unused.
+
+=item OSSL_STORE_ctrl, OSSL_STORE_do_all_loaders, OSSL_STORE_LOADER_get0_engine,
+OSSL_STORE_LOADER_get0_scheme, OSSL_STORE_LOADER_new,
+OSSL_STORE_LOADER_set_attach, OSSL_STORE_LOADER_set_close,
+OSSL_STORE_LOADER_set_ctrl, OSSL_STORE_LOADER_set_eof
+OSSL_STORE_LOADER_set_error, OSSL_STORE_LOADER_set_expect,
+OSSL_STORE_LOADER_set_find, OSSL_STORE_LOADER_set_load,
+OSSL_STORE_LOADER_set_open, OSSL_STORE_LOADER_set_open_ex,
+OSSL_STORE_register_loader, OSSL_STORE_unregister_loader and OSSL_STORE_vctrl
+
+These functions helped applications and engines to create loaders for
+schemes they supported.  These are all deprecated and discouraged in favour of
+provider implementations, see L<provider-storemgmt(7)>.
+
+=item PEM_read_DHparams, PEM_read_bio_DHparams, 
+
+=item PEM_read_DSAparams PEM_read_bio_DSAparams
+
+=item PEM_read_DSAPrivateKey, PEM_read_DSA_PUBKEY, PEM_read_bio_DSAPrivateKey and PEM_read_bio_DSA_PUBKEY
+
+=item PEM_read_ECPKParameters, PEM_read_ECPrivateKey, PEM_read_EC_PUBKEY,
+PEM_read_bio_ECPKParameters, PEM_read_bio_ECPrivateKey and PEM_read_bio_EC_PUBKEY
+
+=item PEM_read_RSAPrivateKey, PEM_read_RSA_PUBKEY, PEM_read_RSAPublicKey,
+PEM_read_bio_RSAPrivateKey, PEM_read_bio_RSA_PUBKEY and PEM_read_bio_RSAPublicKey
+
+=item PEM_write_bio_DHparams, PEM_write_bio_DHxparams, PEM_write_DHparams and PEM_write_DHxparams
+
+=item PEM_write_DSAparams, PEM_write_DSAPrivateKey, PEM_write_DSA_PUBKEY,
+PEM_write_bio_DSAparams, PEM_write_bio_DSAPrivateKey and PEM_write_bio_DSA_PUBKEY
+
+=item PEM_write_ECPKParameters, PEM_write_ECPrivateKey, PEM_write_EC_PUBKEY
+PEM_write_bio_ECPKParameters, PEM_write_bio_ECPrivateKey and PEM_write_bio_EC_PUBKEY
+
+=item PEM_write_RSAPrivateKey, PEM_write_RSA_PUBKEY, PEM_write_RSAPublicKey,
+PEM_write_bio_RSAPrivateKey, PEM_write_bio_RSA_PUBKEY and PEM_write_bio_RSAPublicKey.
+
+See L</Deprecated low-level key reading and writing functions>
```

Comment:
Lots of items without text.  Consistency would suggest merging the items or duplicating the text.